.github/workflows/CI.yml

```
@@ -0,0 +1,81 @@
+name: CI
+
+# Trigger the workflow on push or pull request
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+    name: ${{ matrix.test-suites }} - ${{ matrix.branch }} - ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        test-suites: ["testinstall"]
+        branch: ["master", "stable-4.11", "stable-4.10"]
```

I would suggest to put this onto multiple lines, as in the old Travis setup, so that adding new GAP versions has a minimal diff
```suggestion
        branch:
          - master
          - stable-4.11
          - stable-4.10
```